### DISCUSSION TITLE
Add recommendations to Readme for saving video output on a network share

### DISCUSSION THREAD

**Discussion by spotlesscoder** [bluenviron/mediamtx#1445] [Questions]
## Describe the feature

I am planning to store all video from rtsp-simple-server on a network share.
I am not yet sure whether that is a good idea at all.

Obviously - it might be a problem when the network connection between the host of RTSP Simple Server and the Network Share goes down. Maybe there are some things I can pay attention to during the setup to avoid some problems in the first place?

I guess that might be interesting for others, too

**Comment by aler9** [comment]
Hello, this is a question that involves a process in which the server is just a small part. Network shares are usually set up by using an NFS server. In Linux, you can mount a NFS server folder by inserting a line into /etc/fstab. In this line, you can set whether to wait indefinitely when the connection between you and the server is not available, or to start dropping IO operations. All options are listed here:

https://linux.die.net/man/5/nfs

[a comment by an automated account is not shown]
